app/requirements.txt

```diff
@@ -4,6 +4,7 @@ django-filter==0.11.0
 django-extensions==1.6.1
 django-cors-headers==1.0.0
 google-api-python-client==1.5.0
+oauth2client==2.0.1
```

Comment:
Apparently `oauth2client` used to be a submodule of `google-api-python-client`, but now isn't even a dependency. Not sure how it got in my docker container without an explicit install then.